deploy/redis-writer.yaml

```diff
@@ -0,0 +1,253 @@
+# Copyright 2025 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: redis-writer
+objects:
+# TODO: Add the cache-writer deployment + services
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: redis-writer
+    annotations:
+      ignore-check.kube-linter.io/minimum-three-replicas: "This app doesn't have that much traffic"
+  spec:
+    inMemoryDb: true
+    envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ccx
+    deployments:
+      - name: db-writer
+        minReplicas: ${{CACHE_WRITER_REPLICAS}}
+        webServices:
+          public:
+            enabled: false
+          private:
+            enabled: false
+          metrics:
+            enabled: true
+        podSpec:
+          env:
```

Comment:
I would prefer to create a ConfigMap with the configuration from the parameter rather than setting all these many env vars.

Reply:
@ikerreyes we can also add this to the cache-writer.yaml (https://github.com/RedHatInsights/insights-results-aggregator/pull/2182/files#r2075573069)